sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/package.scala

```diff
@@ -101,7 +101,7 @@ package object expressions  {
       StructType(attrs.map(a => StructField(a.name, a.dataType, a.nullable, a.metadata)))
     }
 
-    // It's possible that `attrs` is a linked list, which can lead to bad O(n^2) loops when
```

Comment:
Could you tell us why you think this comment is error? This comments came from [SPARK-15764 Replace N^2 loop in BindReferences](https://github.com/apache/spark/pull/13505) to explain the problamatic situation.

Reply:
The cost time to access the linked list using ordinal is O (n), but I do not combine BindReferences to analyze the cost time.

Reply:
Ya. The original sentence has a different context at that time.